winget-install.ps1

```diff
@@ -74,7 +74,7 @@ function Start-Init {
     }
 
     #Log file
-    $Script:LogFile = "$LogPath\install.log"
+    $Script:LogFile = "$LogPath\install_$env:UserName.log"
 

```

Comment:
```
if ([System.Security.Principal.WindowsIdentity]::GetCurrent().IsSystem) {
    $Script:LogFile = "$LogPath\install.log"
}
else {
    $Script:LogFile = "$LogPath\install_$env:UserName.log"
}
```

I would prefer something like this then.

Reply:
thanks, good idea. Updated it with the latest commit